contribution-guide.md

```diff
@@ -0,0 +1,85 @@
+ - Open the terminal and navigate to the directory where you wish to clone the project using   
+    `cd <Address-to-the-directory>`
+ - Clone the project and navigate to working directory by typing the commands    
+`git clone https://github.com/<github_account_name>/wayback-machine-chrome.git`   
+ `cd wayback-machine-chrome`
+ - Install the required packages by typing the command   
+`npm install`
+- Now you have a `node_modules` directory populated with packages
+
+## How to install developer version of the extension?
+Once you have cloned the repository to your local machine(this would work with both the original and the forked repositories, but if you wish to make changes, you need to clone the forked version), you can follow these steps, to run install the extension on your browser
+### Chrome
+-   Navigate to  `chrome://extensions`  in your browser. You can also access this page by clicking on the Chrome menu on the top right side of the Omnibox, hovering over  **More Tools**  and selecting  **Extensions**.
+-   Turn on the switch next to  **Developer mode**.
+-   Click the  **Load unpacked**  button and select the  `wayback-machine-chrome`  directory that contains this code.
+
+See  [What are extensions?](https://developer.chrome.com/extensions)  for more information on Chrome extensions.
+
+### Firefox
+
+-   Open Firefox and navigate to  `about:debugging`  in the browser.
+-   Click  **This Firefox**.
+-   Click  **Load Temporary Add-on...**
+-   Open the  `wayback-machine-chrome`  directory and select any file.
+
+See  [Temporary installation in Firefox](https://extensionworkshop.com/documentation/develop/temporary-installation-in-firefox/)  for more information on Firefox extensions.
+## How to contribute code and docs?
+### Find / Create an Issue to work on
+
+ - Explore the [issues section](https://github.com/internetarchive/wayback-machine-chrome/issues) on the repository.
+ - Go through open issues, read comments, talk to people and finalize an issue, you can work on, and get yourself assigned to the issue
+ - Alternatively, if you notice any bug, or have any feature idea, you can create an issue, and discuss with code maintainers, and get yourself assigned to the issue.
+### Write the code/docs
+ - Head over to the forked repository.
+ - Open an existing file, or create a new file.
+ - Make changes and save the files
+ - To write tests for `example.js`, create a new file in the test directory as `example.spec.js`
+ - Run tests by typing  
+`npm test`
+### Push the changes to Github
+ - Open your git terminal
+ - Create a new branch for every feature you add, or every set of bugs you fix and more to that branch using  
+`git branch <branch-name>` (Please use meaningful branch names)  
+`git checkout <branch-name>`
+ - Now add the files which you have made changes to using  
+`git add <filename1> <filename2>` or `git add -A` for all the changed files
+ - Create a commit and write a meaningful commit message using  
```

Comment:
To learn how to write ggod commit message, please see https://chris.beams.io/posts/git-commit/.
You can add here !Thanks